libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { Flex, Image, Stack, Heading, Text } from '@chakra-ui/react'
+
+export const ResponsiveCard = () => {
+  const randomPic = 'https://picsum.photos/600/400'
+
+  return (
+    <Flex
+      boxShadow="sm"
+      bg="white"
+      p={10}
+      gap={8}
+      borderRadius="8px"
+      flexDirection={['column', null, 'row']}
+      m={5}
+    >
+      <Stack>
+        <Image
+          w={['300px', null, 'full']}
+          h={['300px', null]}
```

Comment:
```suggestion
          h={['300px', null, 'full']}
```

Wouldn't it be better if the image always fills the space vertically?